tests/unit/chat_engine/test_chat_engine.py

```diff
@@ -130,18 +130,28 @@ def test_chat(self, namespace, history_length=5, snippet_length=10):
             system_prompt=expected['prompt'],
             context=expected['context'],
             chat_history=messages,
-            max_tokens=200,
             stream=False,
-            model_params=None
+            model_params={'max_tokens': 200}
         )
 
         assert response == expected['response']
 
     @pytest.mark.parametrize("namespace", [
         None, TEST_NAMESPACE
     ])
+    @pytest.mark.parametrize("allow_model_params_override,params_override",
```

Comment:
@malexw I've removed the copy-pasted test cases, making the existing test parameterized instead.